thesis/01-vision.tex

```diff
@@ -1,17 +1,19 @@
 \subsection{Problem characteristics}
-Origami has been around for a long time. It originated in China and Japan, roughly at the same time, from where it 
-spread all around the world\cite{wiki:history-of-origami}. 
-Origami is recognized as the art of paper folding.
+Origami has been around for a long time. 
+The word itself if formed of two parts - \textit{ori} meaning folding, and \textit{kami} meaning paper. 
+The earliest Japanese origami are dated back to the 6th century when the Buddhist monks introduced paper to Japan. \cite{wiki:history-of-origami}
```

Comment:
Hmm, I think we should link to the papers wikipedia references. Not the wikipedia itself.

Reply:
Why? We've taken this from wikipedia, not from external resources so I think it's correct to cite wikipedia, even if it is not a very "scientific" source. Besides that, which reference should we cite? There is plenty of that, and some of it is hard to put a hand on to even search for the part we would need.

Reply:
Because it's advised against. https://en.wikipedia.org/wiki/Wikipedia:Academic_use

The wikipedia article links to The Origami Bible, which is available here: https://archive.org/details/NickRobinsonTheOrigamiBible/page/n11/mode/2up

Reply:
Changed into altogether.